easybuild/easyconfigs/o/ORCA/ORCA-6.0.0-avx2-gompi-2023b.eb

```diff
@@ -0,0 +1,23 @@
+name = 'ORCA'
+version = '6.0.0-avx2'
```

Comment:
`-avx2` is better suited as a `versionsuffix`; the version is still 6.0.0

Reply:
Excellent. I have done this now and recommitted

Reply:
thanks a lot. please also rename the file accordingly to `ORCA-6.0.0-gompi-2023b-avx2.eb`